apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -776,22 +786,35 @@ def _build_tiledb_schema(
 
     dom = tiledb.Domain(dims, ctx=context.tiledb_ctx)
 
+    attrs = []
     enums = []
-    if enumerations is not None:
-        for enum_name in enumerations:
+    metadata = schema.metadata or {}
+    for pa_attr in schema:
+        attr_name = pa_attr.name
+
+        if attr_name in index_column_names:
+            continue
+
+        has_enum = pa.types.is_dictionary(pa_attr.type)
+
+        if has_enum:
+            pa.types.is_integer(pa_attr.type.value_type)
```

Comment:
@nguyenv it looks like this had been

```
something = pa.types.is_integer(pa_attr.type.value_type)
```

but then `ruff` saw the `something` was unused & removed it

anyway, i don't know if you want to omit this line entirely, or assign & use the return value, but just heads-up either way (i just happened to notice this while reading)

Reply:
Thanks for catching that. I have now removed it. 

I don't know how I didn't catch this last week, but I'm still having issues with extending Boolean types. What really weird is that if I use `int64` as the dictionary value in schema and write the values as `int64` instead of `bool`, it works. That doesn't make any sense though.